src/pymovements/utils/aois.py

```diff
@@ -28,7 +28,7 @@
 import polars as pl
 from deprecated.sphinx import deprecated
 
-from pymovements.stimulus import TextStimulus
+import pymovements as pm
```

Comment:
this is necessary to prevent a circular import.

`stimulus.text` imports from `utils.plotting`.
importing `stimulus.text` from `utils.aois`  creates a circular dependency via the `utils` parent module.

as the `utils.checks` module was imported by a lot of other modules, it probably obscured the circular dependency but I'm not sure about this.

anyway, this problem is completely resolved in the follow-up PR #1081 which gets rid of the last `utils` imports.